sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/CustomEvents/ApplicationInsightsEventLogger.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.Logging;
+
+namespace Azure.Monitor.OpenTelemetry.Events
+{
+    /// <summary>
+    /// Default logger for logging Application Insights custom events.
+    /// </summary>
+    public sealed class ApplicationInsightsEventLogger : IApplicationInsightsEventLogger
```

Comment:
Users typically have other logger providers registered as well. Can you comment how this events will look like in the other providers? Since `Critical` severity is used, it'd start showing up by default in other places.

Reply:
> Can you comment how this events will look like in the other providers

Are you asking to clarify that in summary?. They will just look like any other log with `critical` severity.

Reply:
I am not sure yet... People may get shocked seeing Critical logs all of a sudden showing up in their other LogProviders (like ConsoleProvider which is enabled by default).

Reply:
Agree, that is an issue. An alternate option is to use informational and document that users need to allow informational logs from `Azure.Monitor.OpenTelemetry.CustomEvents` category.

Reply:
@vishweshbankwar what if the Exporter creates an extra LoggerFactory that only sends CustomEvents to Azure Monitor?

Reply:
^ That won't meet our needs, as it won't play with LogRecordProcessor and other things in the pipeline, so existing pipeline (LoggerFactory) should be used

Reply:
_An alternate option is to use informational and document that users need to allow informational logs from Azure.Monitor.OpenTelemetry.CustomEvents category (that is not perfect either)._

Can we inject such a filter automatically?

Reply:
No, we need access to IloggingBuilder for that.

Reply:
don't we get access to it from the service collection ?

Reply:
If we rely on service collection, classic .NET framework may not be able to take advantage of this feature. Based on our understanding customEvents are used mostly in classic applications.

Reply:
> An alternate option is to use informational and document that users need to allow informational logs from Azure.Monitor.OpenTelemetry.CustomEvents category (that is not perfect either).

We could achieve this with the distro and the anticipated new UseAzureMonitorExporter API.

Reply:
I changed it to information. Applied the filters for distro users. Will add documentation for users creating the logger instance manually.